DEFAULT_CONFIG.json5

```diff
@@ -175,7 +175,30 @@
   //      ],
   //    },
   //  ],
-
+  //  /// configure acl rules 
+  //  acl: {
```

Comment:
Just realized that `acl` maybe not clear for the user, what about to give more meaningful name like `access_control` ?